code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -576,6 +576,16 @@
 	return TRUE
 
 /obj/structure/closet/container_resist_act(mob/living/user)
+	if(istype(loc, /obj/structure/bodycontainer))
```

Comment:
i wanted to do this with a for loop and a list of possible types to make this smaller, and easier to expand BUUUUUTT the only way i could really do that is with `text2path` and someone told me not to use it

Reply:
What? No you don't, just reference the typepaths directly

Reply:
doesnt matter i used a different method